src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/jpa/entity/Intervention.kt

```diff
@@ -0,0 +1,40 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity
+
+import org.hibernate.annotations.CreationTimestamp
+import java.time.OffsetDateTime
+import java.util.UUID
+import javax.persistence.Entity
+import javax.persistence.EnumType
+import javax.persistence.Enumerated
+import javax.persistence.GeneratedValue
+import javax.persistence.Id
+import javax.persistence.JoinColumn
+import javax.persistence.JoinTable
+import javax.persistence.ManyToMany
+import javax.persistence.ManyToOne
+
+enum class AgeGroup {
+  EIGHTEEN_TO_25, OVER_25
+}
+
+@Entity
+data class Intervention(
```

Comment:
Sorry for the late review.

"Intervention" will be a confusing term because it can mean a lot of things, depending on what is happening: it can mean

- overall idea: "we use interventions to reduce reoffending"
- specific offering: "we offer these interventions"
- what is happening: "I am referring you to this intervention"

The meaning of the word depends heavily on who is speaking and point of time.

This entity is meant to describe the service offered by Dynamic Framework providers, right?

> This "thing" is offered in "these regions" and covers "this category".

We have called this previously as service configuration or contract scopes. Can we call these `ProviderContract` or `ProviderScope` or similar?

Reply:
to answer just the last part of the question - it's my understanding that a provider can offer a service in the same service category for different groups of SUs, i.e. women specific accommodation service in west yorks or 18-25 ETE service in south wales.